### PR TITLE
Feature/client login

### PR DESCRIPTION
**Pull request by dylanlrrb**

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<htlm>
```
Comment (kbchun) [inline client/dist/index.html]: html*

File: client/dist/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+   <meta charset='utf-8'>
+  <title></title>
+</head>
+
+<body>
+	<div id='app'></div>
+
+	<script src="bundle.js"></script>
+</body>
+</htlm>
```
Comment (kbchun) [inline client/dist/index.html]: html*

File: client/dist/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<htlm>
+<language='en'>
+<head>
+   <meta charset='utf-8'>
```
Comment (kbchun) [inline client/dist/index.html]: indentation

File: client/src/components/Action.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return (
+      <div>
+      Action
+      </div>
+    )
```
Comment (kbchun) [inline client/src/components/Action.jsx]: semicolon

File: client/src/components/AddAction.jsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    return (
+      <div>
+        <div>
+        Add Action
+        </div>
+        <div onClick={()=>{console.log('authenticate evernote')}}>
+        Evernote
+        </div>
+      </div>
+    )
```
Comment (kbchun) [inline client/src/components/AddAction.jsx]: semicolon

File: client/src/components/AddConcoctionView.jsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  render() {
+    return (
+      <div>
+        <Trigger trigger={this.state.trigger} servicesDetail={this.props.appState.servicesDetail} />
+        { this.state.actions.map(function(action) {
+          return (
+            <div>
+              <Action action={action} />
+            </div>
+          )
```
Comment (kbchun) [inline client/src/components/AddConcoctionView.jsx]: semicolon

File: client/src/components/AddConcoctionView.jsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        })}
+        <AddAction />
+        <CancelNewConcoction changeViewTo={this.props.changeViewTo} />
+        <SaveNewConcoction />
+      </div>
+    )
```
Comment (kbchun) [inline client/src/components/AddConcoctionView.jsx]: semicolon

File: client/src/components/Concoction.jsx [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+        </div>
+        <div onClick={()=>{this.transitionToEdit()}}>
+          Edit
+        </div>
+      </div>
+    )
+  }
+}
+
+window.Concoction = Concoction;
```
Comment (kbchun) [inline client/src/components/Concoction.jsx]: is this needed if you're exporting this class in line 3?

File: client/src/components/ConcoctionView.jsx [hunk 9]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+class ConcoctionView extends React.Component {
```
Comment (kbchun) [inline client/src/components/ConcoctionView.jsx:3]: export default?

File: client/webpack.config.js [hunk 10]
```diff
@@ -0,0 +1,27 @@
+var webpack = require('webpack');
```
Comment (kbchun) [inline client/webpack.config.js:1]: maybe const/let for the variables in this file to stay consistent with the other client files

**Comment by dylanlrrb** [comment]
all the syntax errors fixed.

**Pull request merged**